components/filter/filter.js

```diff
@@ -222,9 +239,23 @@ class Filter extends LocalizeCoreElement(RtlMixin(LitElement)) {
 			</d2l-input-search>
 		`;
 
```

Comment:
The select all checkbox and summary hides when we search, and is disabled if we're loading or there are no values.  When there are no values selected, we want it to say "Select All" instead of the default "0 selected".